amd_openvx_extensions/CMakeLists.txt

```diff
@@ -63,7 +80,24 @@ if(GPU_SUPPORT AND OpenCL_FOUND AND BUILD_OPENCL)
         message("-- ${Red}WARNING:FFMPEG Not Found -- amd_media module excluded${ColourReset}")
     endif(FFMPEG_FOUND)
 else()
-    message("-- ${Red}WARNING:OpenCL Not Found -- amd_rpp, amd_loomsl, & amd_media modules excluded${ColourReset}")
+    message("-- ${Red}WARNING:GPU support/OpenCL/FFMPEGNot Found -- amd_loomsl, & amd_media modules excluded${ColourReset}")
+endif()
+
+if(GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP")
+        if(HIP_FOUND)
```

Comment:
no need to have HIP_FOUND here if we check it in the above line as I mentioned in my previous comment.

Reply:
Done